library/azure_rm_postgresqlserver.py

```diff
@@ -56,6 +56,20 @@
         description:
             - The maximum storage allowed for a server.
         type: int
+    geo_redundant_backup:
+        description:
+            - Choose between locally redundant(default) or geo-redundant backup
+        type: bool
+        default: False
```

Comment:
Do we need default value here? Because it will send this parameter if you don't provide this field. However, most of the time you just want this parameter unchanged.

Reply:
version 2.9 should be added.

Reply:
Hi, sorry for the delay.
About the parameter, the default behavior of Azure is to set it to False. So I though it woud help the end user of this lib to add it.
I can remove it if you prefer.

I'll add the Ansible version.

Reply:
Sorry for the delay due to low bandwidth. The default value will cause unexpected error when users don't want to update this parameter.